pkgs/development/python-modules/nbsphinx/default.nix

```diff
@@ -30,7 +30,7 @@ buildPythonPackage rec {
   ];
 
   checkPhase = ''
-    ${python.interpreter} -m nbsphinx
+    JUPYTER_PATH=${nbconvert}/share/jupyter ${python.interpreter} -m nbsphinx
```

Comment:
Can we use pythonImportsCheck with preCheck here?

Reply:
Apparently not? It seems like an `export XYZ=123` in `checkPhase` doesn't change the environment for `pythonImportsCheckPhase`.

Reply:
I can change it to
```
  checkPhase = ''
    export JUPYTER_PATH=${nbconvert}/share/jupyter
  '';
  pythonImportsCheck = [
    "nbsphinx"
  ];
```, which does work.